platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/__init__.py

```diff
@@ -0,0 +1,2 @@
+__all__ = ["platform", "chassis", "sfp", "eeprom", "component", "psu", "thermal", "fan", "fan_drawer"]
+from sonic_platform import *
```

Comment:
This line is causing two LGTM alerts ("import * pollutes namespace" and "sonic_platform imports iteself"). Can you please look into fixing these? Is this line even necessary?

Reply:
Addressed this issue. All the lgtm alerts are fixed.